src/deck.jsx

```diff
@@ -49,37 +50,56 @@ class Deck extends React.Component {
   }
   _handleKeyPress(e) {
     const event = window.event ? window.event : e;
-    if (event.keyCode === 37) {
+    if (event.keyCode === 37) { // left arrow
       this._prevSlide();
     }
-    if (event.keyCode === 39) {
+    if (event.keyCode === 39) { // right arrow
       this._nextSlide();
     }
+    if (event.keyCode === 79) { // o
+      this._toggleOverviewMode();
+    }
+    if (event.keyCode === 80) { // o
+      this._togglePresenterMode();
+    }
+  }
+  _toggleOverviewMode() {
```

Comment:
Not sure how to do it. What we really want is not clearing totally the store but bypassing the `Appear` fragment: if i'm in a middle of a slide with some Appears, going into overview mode and back again should keep the Appears already revealed